composer.json

```diff
@@ -15,31 +15,38 @@
     "config": {
         "platform": {
             "php": "7.3"
+        },
+        "allow-plugins": {
+            "typo3/cms-composer-installers": true,
+            "typo3/class-alias-loader": true,
+            "helhum/typo3-console-plugin": true
         }
     },
     "require": {
-        "helhum/typo3-console": "^5.5.5",
-        "typo3/minimal": "^9.5",
-        "typo3/cms-about": "^9.5",
-        "typo3/cms-adminpanel": "^9.5",
-        "typo3/cms-belog": "^9.5",
-        "typo3/cms-beuser": "^9.5",
-        "typo3/cms-felogin": "^9.5",
-        "typo3/cms-fluid-styled-content": "^9.5",
-        "typo3/cms-form": "^9.5",
-        "typo3/cms-impexp": "^9.5",
-        "typo3/cms-info": "^9.5",
-        "typo3/cms-redirects": "^9.5",
-        "typo3/cms-reports": "^9.5",
-        "typo3/cms-rte-ckeditor": "^9.5",
-        "typo3/cms-setup": "^9.5",
-        "typo3/cms-seo": "^9.5",
-        "typo3/cms-sys-note": "^9.5",
-        "typo3/cms-t3editor": "^9.5",
-        "typo3/cms-tstemplate": "^9.5",
-        "typo3/cms-viewpage": "^9.5",
-        "typo3/cms-lowlevel": "^9.5",
-        "slub/slub-web-kartenforum": "dev-master"
+        "helhum/typo3-console": "^6.0.0@beta",
+        "typo3/minimal": "^10.4",
```

Comment:
"`^10.4`" is the correct version constrain for TYPO3 10.4 LTS. In slub_web_kartenforum this setting was not correct ("`10.4`"), why the whole project loaded TYPO3 10.4.0 instead of 10.4.13.
In rare cases, the patch level number may break something. But in this case, everything seems to be perfect.

Reply:
The adjustments are applied [here](https://github.com/slub/slub_web_kartenforum/blob/develop-refactoring-vkf/composer.json). This should resolve this issue.